### PR TITLE
DO-162: Added 2 more countries in the list.

### PR DESCRIPTION
**Pull request by dhruvlahoti**

### CODE REVIEW

File: src/constants/common/commonConstants.js [hunk 1]
```diff
@@ -137,6 +137,8 @@ export const COUNTRIES_FOR_PHONE_INPUT = [
   'ES', // Spain
   'ID', // Indonesia
   'NL', // Netherlands
+  'TH', // Thailand
+  'SA', // Saudi Arabia
 ];
```
Comment (ayushmalik2110) [inline src/constants/common/commonConstants.js:142]: Can we move this constants form Js to Json. So If anything is changed at run time we do not have to restart the server.
Implementation is already completed for this functionality.  You just have to move the code fragment and use reducer.

Comment (dhruvlahoti) [reply src/constants/common/commonConstants.js]: Done

File: config/tenant/default/constants/constants.json [hunk 2]
```diff
@@ -13,5 +13,30 @@
     "CONFIRM_PASSWORD_NOT_MATCH_ERROR_MESSAGE": "Confirm Password does not match.",
     "VALID_PHONE_NUMBER_ERROR_MESSAGE": "Please enter a valid Phone Number",
     "ZIP_CODE_ERROR_MESSAGE": "Zip code must be of 6 digits and do not have any characters."
-  }
+  },
+  "countriesForPhoneInput": [
+    "IN",
```
Comment (ayushmalik2110) [inline config/tenant/default/constants/constants.json:18]: "_NOTE_": "Please manage this file keys in alphabetical order". 
I thik 'c' comes before 'l'. : )

Comment (dhruvlahoti) [reply config/tenant/default/constants/constants.json:18]: The library automatically makes it in alphabatical order, but let me know if it is still required to do in this json file.

Comment (ayushmalik2110) [reply config/tenant/default/constants/constants.json:18]: I am not talking about the country list. I am talking about the whole json file.
{
 logoutInfoMessages: {},
 validationMessages: {},
 countriesForPhoneInput: {}
}

Comment (dhruvlahoti) [reply config/tenant/default/constants/constants.json]: Done, Sir

**Pull request merged**